error-prone-contrib/src/main/java/tech/picnic/errorprone/refastertemplates/ReactorTemplates.java

```diff
@@ -271,6 +310,43 @@ Flux<S> after(Flux<T> flux) {
     }
   }
 
+  /**
+   * Prefer {@link Flux#concatMapIterable(Function)} over alternatives that require an additional
+   * subscription.
+   */
+  static final class ConcatMapIterableIdentity<S> {
```

Comment:
We use `S` when we need two generics, in this case we can simply use `<T>`.

Reply:
Ah yes :). (Also something to auto-unify on day :see_no_evil:.)